src/modules/EventsList/EventsList.jsx

```diff
@@ -0,0 +1,55 @@
+import './EventsList.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+import { StaggeredMotion, spring } from 'react-motion'
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import EventTile from 'components/EventTile';
+
+@cxHelpers("EventsList")
+class EventsList extends PureComponent {
+  static propTypes = {
+    isLoading: PropTypes.bool.isRequired,
+    events: PropTypes.array
+  };
+
+  renderEvents(events) {
+    const defaultStyles = events.toJS().map(e => ({s: 0}));
+    const styles = (prevInterpolatedStyles) => prevInterpolatedStyles.map((_, i) => {
```

Comment:
This and the function below could be their own function blocks.  It makes "renderEvents" a little thicker than it needs to be to have two distinct funcitons defined in it before calling the return block